doc/CONTRIBUTING.md

```diff
@@ -60,8 +60,8 @@ called `npm` is installed as well.
 
 Not all code contributions will be accepted. The safest way to ensure
 that you don't waste time working on something that gets rejected is to
-run your idea(s)/plan(s) past the developpers first.
```

Comment:
If this line is changed, I would consider if there's a better alternative for `idea(s)/plan(s)` .  `ideas` maybe is enough alone.